apollo-router/src/query_planner/bridge_query_planner_pool.rs

```diff
@@ -120,8 +121,11 @@ impl tower::Service<QueryPlannerRequest> for BridgeQueryPlannerPool {
         &mut self,
         _cx: &mut std::task::Context<'_>,
     ) -> std::task::Poll<Result<(), Self::Error>> {
-        // todo: check if the queue is empty
-        std::task::Poll::Ready(Ok(()))
+        if self.sender.is_full() {
```

Comment:
@o0Ignition0o I put pending here for now but it may be wiser to error here and backpressure up to the client. Let's discuss that